.storybook/main.js

```diff
@@ -1,4 +1,71 @@
+const path = require('path');
+
 module.exports = {
-  "stories": ["../src/**/*.stories.@(tsx)"],
-   "addons": ["@storybook/addon-essentials"]
-}
+	stories: ['../src/**/*.stories.@(tsx)'],
+	addons: ['@storybook/addon-essentials'],
+	webpackFinal: async (config) => {
```

Comment:
As recommended by https://storybook.js.org/docs/react/configure/webpack moved from webpack to main.js